src/translations/maps/baseMap.js

```diff
@@ -0,0 +1,150 @@
+import { rangeEach } from '../../helpers/number';
+import { mixin } from '../../helpers/object';
+import { isFunction } from '../../helpers/function';
+import localHooks from '../../mixins/localHooks';
+
+/**
+ * Map for storing mappings from an index to a value.
```

Comment:
`Map for storing mappings` sounds vague... This is a base class, so I'd rather write something like `Map of an array index to any other value`

Reply:
For me it's ok. We will discuss that.

Reply:
For me this is not, ok, because this description doesn't say anything more than the property name. I would expect to learn from the description what is meant by "sequence of indexes" and what is the purpose of it.